### ISSUE TITLE
[golangci-lint] error scenario handling

### ISSUE THREAD

**Issue by CarlJi** [qiniu/reviewbot#144]
```bash
[reviewbot-5cf968b686-88rt8] 2024/05/31 10:16:23.895731 [cc9ad950-1f36-11ef-91a2-3efc97036417][INFO] server.go:138: [golangci-lint] config on repo qbox/configcenter: Linter{Enable: true, WorkDir: /var/tmp/gitrepo1970249307, Command: golangci-lint, Args: [run -D staticcheck], ReportFormat: github_check_run}
[reviewbot-5cf968b686-88rt8] 2024/05/31 10:16:24.073542 [cc9ad950-1f36-11ef-91a2-3efc97036417][ERROR] linters.go:127: golangci-lint run failed: exit status 3, mark and continue
[reviewbot-5cf968b686-88rt8] 2024/05/31 10:16:24.073633 [cc9ad950-1f36-11ef-91a2-3efc97036417][WARN] linters.go:250: unexpected linter check output: level=error msg="Running error: context loading failed: failed to load packages: failed to load packages: failed to load with go/packages: err: exit status 1: stderr: go: go.mod requires go >= 1.22.3 (running go 1.21.5; GOTOOLCHAIN=local)\n"
```

**Comment by CarlJi** [comment]
another：

```bash
2024/05/31 10:31:37.059317 [bc881800-1f38-11ef-8ab6-f109983b1467][INFO] server.go:138: [golangci-lint] config on repo qbox/net-deftones: Linter{Enable: t
rue, WorkDir: /var/tmp/gitrepo3932887171, Command: golangci-lint, Args: [run -D staticcheck], ReportFormat: github_check_run}
2024/05/31 10:31:37.201228 [bc881800-1f38-11ef-8ab6-f109983b1467][ERROR] linters.go:127: golangci-lint run failed: exit status 3, mark and continue
2024/05/31 10:31:37.201345 [bc881800-1f38-11ef-8ab6-f109983b1467][WARN] linters.go:250: unexpected linter check output: level=error msg="Running error: context loading failed: failed to load packages: failed to load packages: failed to load with go/packages: err: exit status 1: stderr: go: go.mod requires go >= 1.22 (running go 1.21.5; GOTOOLCHAIN=local)\n"
```

**Comment by CarlJi** [comment]
fixed by #142
